linux/build.gradle

```diff
@@ -86,8 +86,11 @@ def getJdkMajorVersion() {
         throw new IllegalArgumentException("Unparsable JDK major version '$versionString'")
     }
     int version = Integer.valueOf(versionString)
-    if (version < 8) {
-        throw new IllegalArgumentException("JDK major version '$version' out of range")
+
+    // Do not change the numbers without updating the metadata declarations
+    // (dependencies, jinfo priorities)!
+    if (version < 8 || version > 12) {
```

Comment:
Should we be going to 13 now?

Reply:
It‘s prepared but comes in a separate pull request.